editortools/select.py

```diff
@@ -1231,17 +1231,39 @@ def openCommands(self):
         file = open(filename, 'w')
         first = True
         space = config.commands.space.get()
-        sorting = config.commands.sorting.get()
+        from collections import namedtuple
+        SortingTuple=namedtuple('sorting','order invertX invertY invertZ')
+        GroupingTuple=namedtuple('grouping','chains')
+        sorting=SortingTuple(
+            order = config.commands.sorting.get(),
+            invertX = config.commands.invertX.get(),
+            invertY = config.commands.invertY.get(),
+            invertZ = config.commands.invertZ.get()
+        )
+        grouping = GroupingTuple(chains=config.commands.groupchains.get())
         edit = fileEdit(filename, os.path.getmtime(filename), self.editor.selectionBox(), self.editor,
                         self.editor.level)
 
         order = []
-        if sorting == "chain":
+        if grouping.chains:
             skip = []
             done = []
             chainStored = []
             for coords in GetSort(self.editor.selectionBox(), sorting):
-                (x, y, z) = coords
+                if sorting.order == "xyz":
```

Comment:
I think I may have the order backwards. I was very confused on the sort order when writing this. Should sort XYZ currently does ittertool.product(z,y,x). Meaning we move across X axis first, rather than last.

I am pretty sure this is backwards, but when looking at the CB on the screen it's what I seem to think it means.